src/main/java/spotify/api/enums/HttpStatusCode.java

```diff
@@ -1,5 +1,12 @@
 package spotify.api.enums;
 
+/**
+ * This class contains all suitable HTTP Status codes used within the API.
+ *
+ * @author Jiankai Zheng (jk.zheng@hotmail.com)
+ * @since 1.0.0
```

Comment:
This was not added in 1.0.0, I'm not sure which version this was added, you need to look through the verison releases on Github.

Reply:
You say not added in 1.0.0, but I show it was added in 1.0.0? Unless I am reading the releases wrong? I went to Comparing1.0.0...main and saw it there